integrationtests/config/rho_loss.yaml

```diff
@@ -0,0 +1,73 @@
+pipeline:
+  name: RHO-Loss Test Pipeline
+  description: Example pipeline
+  version: 1.0.0
+model:
+  id: ResNet50
```

Comment:
i have not taken a look at this PR, but we cannot train a ResNet50 on a github runner :D if at all, we should use a very very small dummy model, just some linear layer or something, for the integration test

Reply:
thanks for spotting this. I changed it to a smaller one